Automattic/wp-calypso
# Jetpack Cloud: Improve ActivityCard and realtime backup view

#### Changes proposed in this Pull Request

* Updates styles and some form on the ActivityCard component
* Applies styles to the Realtime context for backups
* One easy bug fix to backup detail page

#### Testing instructions

* Load up the backup section for a site with realtime backups enabled.
* Check that styles are applied to the activity cards for a day with activities. They should roughly reflect the Figma design.
* Check that you are able to perform a restore and download a backup for an activity.
* Check that the detail page works.
